tests/system/action/committee/test_update.py

```diff
@@ -132,6 +132,190 @@ def test_update_both_forwarded_and_received(self) -> None:
         )
         self.assert_model_exists("committee/3", {"forward_to_committee_ids": [1]})
 
+    def test_update_complex_1(self) -> None:
+        """A->B exists, test that the request for B with B->C works and sets the reverse relations on A and C correctly"""
```

Comment:
This test is unnecessary since the previous relations are not influenced by the request. I don't know what I was thinking there :D anyway, you can delete this

Reply:
Done.